docs/resources/Channel.md

```diff
@@ -498,6 +498,109 @@ Embed types are "loosely defined" and, for the most part, are not used by our cl
 | type     | integer   | the [type of channel](#DOCS_RESOURCES_CHANNEL/channel-object-channel-types) |
 | name     | string    | the name of the channel                                                     |
 
+### Allowed Mentions Object
+
+The allowed mention field allows for more granular control over mentions without various hacks to the message content. This will always validate against message content to avoid phantom pings (e.g. to ping everyone, you must still have `@everyone` in the message content), and check against user/bot permissions.
+
+###### Allowed Mention Types
+
+| Type              | Value      | Description                           |
+| ----------------- | ---------- | ------------------------------------- |
+| Role Mentions     | "roles"    | Controls role mentions                |
+| User Mentions     | "users"    | Controls user mentions                |
+| Everyone Mentions | "everyone" | Controls @everyone and @here mentions |
+
+###### Allowed Mentions Structure
+
+| Field | Type                           | Description                                                                                                   |
+| ----- | ------------------------------ | ------------------------------------------------------------------------------------------------------------- |
+| parse | array of allowed mention types | An array of [allowed mention types](#DOCS_RESOURCES_CHANNEL/allowed-mention-types) to parse from the content. |
+| roles | list of snowflakes             | Array of role_ids to mention                                                                                  |
+| users | list of snowflakes             | Array of user_ids to mention                                                                                  |
+
+###### Allowed Mentions Reference
+
+Due to the complexity of possibilities, we have included a set of examples and behavior for the allowed mentions field.
+
+If `allowed_mentions` is _not_ passed in (i.e. the key does not exist), the mentions will be parsed via the content. This corresponds with existing behavior.
+
+In the example below we would ping @here (and also @role124 and @user123)
+
+```json
+{
+  "content": "@here Hi there from <@123>, cc <@&124>"
+}
+```
+
+To suppress all mentions in a message use:
+
+```json
+{
+  "content": "@everyone hi there, <@&123>",
+  "allowed_mentions": {
+    "parse": []
+  }
+}
+```
+
+This will suppress _all_ mentions in the message (no @everyone or user mention)
+
+The `parse` field is mutually exclusive with the other fields
+e.g. In the example below, we would ping users `123` and role `124`, but _not_ @everyone
+Note that passing a `Falsy` value ([], null) into the "users" field does not trigger a validation error.
```

Comment:
```suggestion
This will suppress _all_ mentions in the message (no @everyone or user mention).

The `parse` field is mutually exclusive with the other fields. In the example below, we would ping users `123` and role `124`, but _not_ @everyone. Note that passing a `Falsy` value ([], null) into the "users" field does not trigger a validation error.
```